### PR TITLE
Fix Documentation for Gitlab/Jenkins Local Docker Setup

### PR DESCRIPTION
**Pull request by b-fein**
### Description
This fixes the heading styles in the sections following https://artemis-platform.readthedocs.io/en/latest/dev/setup/jenkins-gitlab/#etc-nginx-common-common-ssl-conf

### Screenshots
Before:
Screenshot_20210413_205023
After:
Screenshot_20210413_225934

### CODE REVIEW

File: docs/dev/setup/jenkins-gitlab.rst [hunk 1]
```diff
@@ -1208,7 +1209,7 @@ Jenkins
 .. _artemis-1:
 
 Artemis
--------
+~~~~~~~
 
 1. In ``docker-compose.yml``
```
Comment (dfuchss) [inline docs/dev/setup/jenkins-gitlab.rst:1214]: If we fix the local setup documentation, we could fix some possible issues in this section as well :)

1. I think Point 2.2 is obsolete and can be deleted
2. the space in "range .NetworkSettings" seems to be wrong. Correct version: "range.NetworkSettings"
3. Maybe the profile "scheduling" should be added to point 1.2

**Review by dfuchss — CHANGES_REQUESTED** [review]
In general, I think the changes are good. I would suggest to fix three issues as well in this PR as we deal with the documentation for Gitlab / Jenkins :)

File: docs/dev/setup/jenkins-gitlab.rst (hunk 1 above)
Comment (krusche) [reply docs/dev/setup/jenkins-gitlab.rst:1214]: Good points. Could you directly fix those issues?

Comment (b-fein) [reply docs/dev/setup/jenkins-gitlab.rst:1214]: Done. Point 2.1 was also obsolete and would even lead to a startup failure if applied.

**Review by krusche — APPROVED** [review]

**Pull request merged**